kalite/distributed/features/steps/superuser_create.py

```diff
@@ -46,6 +52,8 @@ def step_impl(context):
 
 @then("the modal won't dismiss")
 def step_impl(context):
+    driver = webdriver.Firefox()
+    driver.implicitly_wait(2)
```

Comment:
Isn't it also problematic to create a new web driver for a single test method? Can't these instances somehow be shared across tests, since the tests are sequentially executed anyways?